javascript/sdk/lib/types.ts

```diff
@@ -18,7 +18,7 @@ export interface Test {
   id: string;
   name: string;
   unit: string;
-  code?: string;
+  alert: string;
```

Comment:
do you still want the question mark?

Reply:
@mireaulf says its not optional just empty (`""`)  when blank